howto/finetune_full.md

```diff
@@ -0,0 +1,104 @@
+# Full Finetuning
+
+Full finetuning updates all layers in the pretrained LLaMA model. This *regular* finetuning procedure is typically considered as the baseline for parameter-efficient alternatives such as Low-Rank Adaptation (LoRA) or LLaMA-Adapter.
+
+The current  [finetune_full.py](../scripts/finetune_full.py) we provide uses 4 A100 GPUs with a fully-sharded data parallel strategy to finetune Lit-LLaMA 7B on [Alpaca](https://github.com/tatsu-lab/stanford_alpaca) dataset. The A100 GPUs have 40 Gb each, but it may require less memory to finetune this model.
```

Comment:
```suggestion
The current  [finetune_full.py](../scripts/finetune_full.py) we provide uses 4 A100 GPUs with a fully-sharded data parallel strategy to finetune Lit-LLaMA 7B on [Alpaca](https://github.com/tatsu-lab/stanford_alpaca) dataset. The A100 GPUs have 40 GB each, but it may require less memory to finetune this model.
```

Reply:
Regarding the comment about memory, I think that if we were training this with the A100, there is probably a bit more room and we can push the `micro_batch_size to > 4` to make efficient use of the GPU.